packages/loader/container-loader/README.md

```diff
@@ -81,6 +87,19 @@ When container is closed, the following is true (in no particular order):
 
 `"closed"` event is available on Container for hosts. `"disposed"` event is delivered to container runtime when container is closed. But container runtime can be also disposed when new code proposal is made and new version of the code (and container runtime) is loaded in accordance with it.
 
+#### `Container.dispose()`
+
+Once disposed, container terminates connection to ordering service, and any local changes (former or future) do not propagate to storage. This method is to be used when the container is **NOT** expected to be used anymore.
+
+When container is disposed, the following is true (in no particular order):
+
+1. Container.closed property is set to true
+2. "disposed" event fires on container
+3. "disconnected" event fires, if connection was active at the moment of container disposal.
+4. "dispose" event fires on container runtime
```

Comment:
Sure. Is that something you think we could knock out before the next major bump?  Like, add a duplicate `disposed` event on ContainerRuntime and then in next remove `dispose`?  Dunno if it would be that simple.  Would be nice to not be in the incongruent state for long.

Reply:
I think it could make it, but we might not want to deprecate something in the last minor before a major release (per Tyler Butler's suggestion)